Holovibes/includes/gui/selections/overlays/composite_area_overlay.hh

```diff
@@ -10,7 +10,7 @@ namespace holovibes::gui
 {
 /*! \class CompositeAreaOverlay
  *
- * \brief #TODO Add a description for this class
+ * \brief class that represents a composite area overlay in the window.
```

Comment:
**C**lass
I think the file's documentation is better fitted here than this sentence, which is more generic